classes/user.php

```diff
@@ -36,6 +36,10 @@ class User extends PlUser
         'x'      => '',
         'master' => 'master.',
         'phd'    => 'doc.',
+        'bachelor' => 'alumni.',
+        'executive' => 'alumni.',
+        'graduate' => 'alumni.',
+        'masterspe' => 'alumni.',
```

Comment:
Stupid question from someone that doesn't really know how platal works: does these extra alumni mean there are 2 alumni entries added in the mail database/webpage? Or alternatively, does the insert email address gets the correct domain ID for the alumni domain given there are multiple alumni domain entries in the email_virtual_domains table?

Maybe this is not where that is even read from, but aside from the INSERT mysql commands I didn't see anywhere else where the domains are e-mail domains defined in the code.